themes/default/content/resources/getting-started-with-iac-aws/index.md

```diff
@@ -44,11 +44,7 @@ hero:
     image: "/icons/containers.svg"
 
 # Webinar pages support multiple session via the 'multiple' property.
-multiple:
-  - datetime: 2022-07-28T10:00:00.000-07:00
-    hubspot_form_id: 21e1d99b-9d59-43cb-a5bb-cb77217db726
-  - datetime: 2022-09-08T10:00:00.00-07:00
-    hubspot_form_id: "99717c20-c713-452a-9ac0-26af767cbf36"
```

Comment:
The Form ID at the bottom of the file needs to be updated to this value. It looks like it is pointing to an old form right now.

Reply:
good catch!